### DISCUSSION TITLE
install minio gateway Docker on Raspberry

### DISCUSSION THREAD

**Discussion by llinuxde** [minio/minio#11661] [Q&A] [answered]
Hi all,

is it possible to install minio gateway Docker on Raspberry? I got this error, when i try to install it:
`docker: no matching manifest for linux/arm/v7 in the manifest list entries.`

**Comment by harshavardhana** [comment, marked as answer]
> Hi all,
> 
> is it possible to install minio gateway Docker on Raspberry? I got this error, when i try to install it:
> `docker: no matching manifest for linux/arm/v7 in the manifest list entries.`

We do not support arm32 on docker, alternatively download the binary and run.
